.github/workflows/ci.yml

```diff
@@ -123,3 +123,26 @@ jobs:
       - run: make checkfast
 
       - run: make testfast
+
+  codecov:
+    name: Code Coverage
+    runs-on: ubuntu-latest
+    continue-on-error: true # well, its nightly and highly experimental
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
+          components: llvm-tools-preview
+          override: true
+
+      - uses: Swatinem/rust-cache@v1
+
+      - uses: Swatinem/fucov@v1
+
+      - uses: codecov/codecov-action@v1
```

Comment:
Please pin this to the current SHA of the action until we approve codecov as trusted party.